doc/source/flexibility/proxy_protocol_support.rst

```diff
@@ -33,7 +38,7 @@ System Variables
   :dyn: No
   :default: ``(empty string)``
 
-This variable is a global-only, read-only variable, which is either a ``*`` (to enable proxying globally, a non-recommended setting), or a list of comma-separated IPv4 and IPv6 network and host addresses, for which proxying is enabled. Network addresses are specified in CIDR notation, i.e. ``192.168.0.0/24``. To prevent source host spoofing, the setting of this variable must be as restrictive as possible to include only trusted proxy hosts.
+This variable is a global-only, read-only variable, which is either a ``*`` (to enable proxy globally, a non-recommended setting), or a list of comma-separated IPv4 and IPv6 network and host addresses, for which proxy is enabled. Network addresses are specified in CIDR notation, i.e. ``192.168.0.0/24``. To prevent source host spoofing, the setting of this variable must be as restrictive as possible to include only trusted proxy hosts.
```

Comment:
I would state the recommended value first and then mention * as an option. And add why it's not recommended using it and when one may use it.

Reply:
Rewritten